ansible/roles/proxy/tasks/main.yml

```diff
@@ -25,3 +25,10 @@
   file: path={{ tarballs_dir }} state=directory
 - name: "copy /etc/hosts to proxy"
   template: src=roles/proxy/templates/etc_hosts dest=/etc/etc_hosts owner=root group=root mode=0644
+# Update ca-certificates to ensure that CA certificates are new enough
+# to support SSL verification against websites used to download software
+- name: "update ca-certificates package to latest version"
+  yum:
+    name: 'ca-certificates*'
```

Comment:
Why the wildcard? That seems questionable. RPMs are named based on artifact name, (epoch), version, and release (followed by arch). For YUM, it is sufficient to install/update by name only. See https://blog.jasonantman.com/2014/07/how-yum-and-rpm-compare-versions/

To install or update the ca-certificates RPM, it is sufficient to specify `ca-certificates`. Adding the wildcard would match on tangentially related packages, perhaps debug packages, or 3rd party packages that start with that pattern. It does help that `update_only` is specified, so you don't install any new packages, but it still implies that this could grab more than intended.